docs/source/module_configuration/staticsite.rst

```diff
@@ -38,6 +38,16 @@ be displayed on stack creation/updates.
 A start-to-finish example walkthrough is available
 in the :ref:`Conduit quickstart<qs-conduit>`.
 
+**Please note:** The CloudFront distribution will take a significant amount
```

Comment:
Should we also include a prompt informing at launch time?

Reply:
Moved this to the `deploy` method as it's not changing the YAML and should probably be out of the `pre_build` hooks.